### ISSUE TITLE
[CTA Section]: QA: Screen reader is not identifying the Microsoft Logo 

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#9107]
**Engineering info:**  
- Innovation Team engineer: 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2525
- DPO consulting engineer: N/A

-----

### Description

Page element:
"Microsoft Logo" which will get by adding the Logo from the Knobs

Issue : Screen reader is not identifying the Mentioned Logo 

Expected: Screen reader is expected to identify  the mentioned Logo With appropriate Alt text for the user.

User Impact: When the screen reader doesn’t identify the page elements, it impacts the experience of users who are blind or have vision impairment. They can miss important information or fail to perform an action on the web page. 

### Component(s) impacted

CTA-Section

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.36.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by oliviaflory** [comment]
@proeung FYI this bug came in for the Logo variant of CTA section
